### ISSUE TITLE
Knowls for Kodaira type

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#5568]
On an elliptic curve page, it would be great if the reduction type could show a picture of the reduction.

**Comment by JohnCremona** [comment]
One suggestion was (for elliptic curves) to have each Kodaira type an object in its own right with its own page, including picture).

**Comment by alozanoroble** [comment]
I agree with @JohnCremona that it would be great to have separate pages for Kodaira type, so that the type in an elliptic curve page links to the Kodaira type, where a picture of the reduction can be found among other things.
